instrumentation/aws-sdk/aws-sdk-2.2/library/src/main/java/io/opentelemetry/instrumentation/awssdk/v2_2/TracingExecutionInterceptor.java

```diff
@@ -137,9 +137,9 @@ private void decorateWithExAttributesData(Span span, ExecutionAttributes attribu
     String awsServiceName = attributes.getAttribute(SdkExecutionAttribute.SERVICE_NAME);
     String awsOperation = attributes.getAttribute(SdkExecutionAttribute.OPERATION_NAME);
 
-    span.setAttribute("aws.agent", COMPONENT_NAME);
-    span.setAttribute("aws.service", awsServiceName);
-    span.setAttribute("aws.operation", awsOperation);
+    span.setAttribute("aws-sdk.agent", COMPONENT_NAME);
```

Comment:
The aws changes will be very breaking for us in the collector. I need to get back to my spec PR for these... Do intend to change these, but need to change the collector handling in a backwards compatible way first.

Reply:
I reverted the aws attribute name changes